hadoop-hdds/docs/content/feature/SCM-HA.md

```diff
@@ -0,0 +1,162 @@
+---
+title: "SCM High Availability"
+weight: 1
+menu:
+   main:
+      parent: Features
+summary: HA setup for Storage Container Manager to avoid any single point of failure.
+---
+<!---
+  Licensed to the Apache Software Foundation (ASF) under one or more
+  contributor license agreements.  See the NOTICE file distributed with
+  this work for additional information regarding copyright ownership.
+  The ASF licenses this file to You under the Apache License, Version 2.0
+  (the "License"); you may not use this file except in compliance with
+  the License.  You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+-->
+
+Ozone has two leader nodes (*Ozone Manager* for key space management and *Storage Container Management* for block space management) and storage nodes (Datanode). Data is replicated between Datanodes with the help of RAFT consensus algorithm.
```

Comment:
"Ozone has two leader nodes" seems confusing to me. Can we call them master nodes(or master services instead)

Reply:
*master* word is controversial due to the `master/slave` meaning. I am not a native English and I am not living in US, my primary association is the [first meaning](https://www.dictionary.com/browse/master) not the second one.

> a person with the ability or power to use, control, or dispose of something

But I tried to avoid any confusions. (I have no idea what are the associations of this word in other regions...)

What about "Ozone has two metadata-manager nodes"?

Reply:
I am fine with "metadata-manager nodes"

Reply:
Yeah, leader is also misleading due to the meaning of RAFT leader... I got it. Fixed in 67c545c5d